### ISSUE TITLE
[Bug] error in file preview

### ISSUE THREAD

**Issue by firecentaur** [Laravel-Backpack/devtools-issues#39]
# Bug report

### What I did

I clicked on "See-files"in devtools

### What I expected to happen

To see the files

### What happened

got a PHP error: 
```
Call to a member function isClass() on array (View: /var/www/<myfolder>/vendor/backpack/devtools/resources/views/widgets/preview-files.blade.php)
```

### What I've already tried to fix it

nothing

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
### LARAVEL VERSION:
v8.83.2@b91b3b5b39fbbdc763746f5714e08d50a4dd7857

### BACKPACK VERSION:
5.0.8@f811634e8c4789fa15026d70592515b8051700ad

### Operating System and Server Setup

I encountered the problem above on my Windows 10, WSL, Apache2, Mysql

**Comment by firecentaur** [comment]
sure, here is a gist of the contents https://gist.github.com/firecentaur/ece708d99f5d4fb92d3c698ed7279811

**Comment by maurohmartinez** [comment]
@tabacitu I am not able to reproduce this bug. I find the error strange since we first check if `$file` is `array`, and only **if not** we then call `->isClass()`. Technically, if it is NOT an array, we should never reach that line, let alone get `Call to a member function isClass() on array` error... :/

Any suggestion how this can be reproduced? Is this still a bug?

**Comment by tabacitu** [comment]
Ok so I played around a bit, and I was able to get the same error ([stack trace here](https://flareapp.io/share/17WbA0DP#F95)):

![CleanShot 2022-07-13 at 11 31 44](https://user-images.githubusercontent.com/1032474/178688426-1112c479-0441-44de-8784-768f73636af0.png)

Not sure if these are the same circumstances, but hey 🤷‍♂️ it's a place to start.

Here's what I did:
- used DevTools to generate a Book migration, model, CRUD;
- renamed the request to `UpdateBookRequest.php`
- duplicated the above to have a `StoreBookRequest.php`
- refreshed the page for "Requests" in Devtools: `admin/devtools/model/14263832/related-files/crud_requests`

So _I think_ there error might show up when DevTools finds multiple files that might be a match? Maybe? 👀
